src/views/Game/Game.tsx

```diff
@@ -1342,6 +1342,7 @@ export function Game(): JSX.Element {
             <span className="play-buttons">
                 <span>
                     {((cur_move_number >= 1 &&
+                        !goban?.current?.engine.rengo &&
```

Comment:
Technically you don't need the first `?.` since `goban` is always initialized, but `.current` can be null so you do need the second `?.`, so we would only need `!goban.current?.engine.rengo`.  To answer your question, yep this is a fine way to guard against the goban ref not being initialized yet.